Remove inappropriate "please", minor rewording (#4249)
iterative/dvc.org: content/docs/user-guide/data-management/importing-external-data.md

```

⚠️ DVC requires both SSH and SFTP access to work with remote SSH locations.

> Don't forget to check that you are able to connect both ways with tools like `ssh` and

Reviewer: ```suggestion
Check that you can connect both ways with tools like `ssh` and
```

$ dvc remote modify --local myssh password 'mypassword'
```

> > You may refer to `dvc remote modify` for more details like setting up access

Reviewer: ```suggestion
> Refer to `dvc remote modify` for more details like setting up access
```
